style.json

```diff
@@ -761,24 +739,24 @@
       "metadata": {"mapbox:group": "101da9f13b64a08fa4b6ac1168e89e5f"},
       "source": "openmaptiles",
       "source-layer": "place",
-      "maxzoom": 14,
+      "minzoom": 5,
+      "maxzoom": 24,
```

Comment:
I would keep maxzoom at 14. Once you are at that zoom level you shouldn't need the city label anymore.